test/jasmine/tests/plot_api_test.js

```diff
@@ -2781,6 +2865,53 @@ describe('Test plot api', function() {
             .then(done);
         });
 
+        it('can change data in candlesticks multiple times', function(done) {
```

Comment:
Should we added add a finance trace case for the [_can redraw with no changes as a noop_](https://github.com/plotly/plotly.js/blob/7d91d29a48bc882afccda635054dfa1a7ece85fd/test/jasmine/tests/plot_api_test.js#L2856) set of tests?

Reply:
> Should we added add a finance trace case for the can redraw with no changes as a noop set of tests?

It's there -> https://github.com/plotly/plotly.js/blob/7d91d29a48bc882afccda635054dfa1a7ece85fd/test/jasmine/tests/plot_api_test.js#L2826
and the previous `fullJson` exemption was cleared in this PR -> https://github.com/plotly/plotly.js/pull/2561/files#diff-5a140ce1cc87a420496039bbf8d699f1L2913